setup.py

```diff
@@ -1,15 +1,24 @@
 from setuptools import setup
 
-setup(name='django-hooked',
+setup(
+    name='django-gapi-hooked',
     version='0.2.0',
     author='G Adventures',
     author_email='software@gadventures.com',
+    description='A tiny library to add a G Adventures API Web Hook receiver view to your code base.',
+    download_url='https://github.com/gadventures/django-gapi-hooked/tarball/0.2.0',
```

Comment:
Version should probably be a variable so it doesn't have to be updated in multiple places.